packages/burner-wallet-connector/vite.config.ts

```diff
@@ -0,0 +1,22 @@
+// vite.config.js
```

Comment:
Does this file is been used? I believe we could remove ti

Reply:
Is used for define dirname and filename because without that i can't use chainConfig path and is used for set jsdom as test environment.